src/lib.rs

```diff
@@ -249,11 +249,15 @@ impl<T: MutEventSubscriber + ?Sized> EventSubscriber for Mutex<T> {
 
 impl<T: MutEventSubscriber + ?Sized> MutEventSubscriber for Mutex<T> {
     fn process(&mut self, events: Events, ops: &mut EventOps) {
-        self.lock().unwrap().process(events, ops);
+        // If another user of this mutex panicked while holding the mutex, then
+        // we terminated the process.
```

Comment:
nit: can we please replace "terminated" with "terminate" here and in the following comment as well?